Sources/Controllers/OARootViewController.m

```diff
@@ -504,6 +506,7 @@ - (BOOL) handleIncomingURL:(NSURL *)url_
     }
     else if ([ext caseInsensitiveCompare:@"osf"] == NSOrderedSame)
     {
+        [_onAllMenusHidingObservable notifyEvent];        
```

Comment:
Observable is not needed. You may access OAMapPanelViewController from here. Implement some method here which will close needed views ( **onHandleIncomingURL:(NSString \*)ext** for example)